### PR TITLE
Bump version to 2.32.1

### LINKED ISSUE #15: libuuid.so.1: no version information available required by .../lib/libdapserver.so.7
This is a copied over issue from conda-forge/gdal-feedstock#200

When I run a tool like`gdalinfo`, I get
```
gdalinfo: /opt/miniconda2/envs/prod/bin/../lib/libuuid.so.1: no version information available (required by /opt/miniconda2/envs/prod/bin/../lib/libdapserver.so.7)
```

This only started after a recent full upgrade of my conda-forge env.
```
$ conda package -w /opt/miniconda2/envs/prod/lib/libuuid.so.1
/opt/miniconda2/envs/prod/lib/libuuid.so.1          defaults::util-linux-2.21-0
/opt/miniconda2/envs/prod/lib/libuuid.so.1          conda-forge::libuuid-1.0.3-1
```
The installation of `libuuid` changes the sym link for `libuuid.so.1` to `libuuid.so.1.0.0`, instead of `libuuid.so.1.3.0` which seems to be the default with a fresh install of conda.  If I manually update that symlink back to `libuuid.so.1.3.0`, the warning message above goes away.

### PR DESCRIPTION
**Pull request by nehaljwani**
Fixes #15

### CODE REVIEW

**Comment by nehaljwani** [comment]
@conda-forge-admin please rerender

**Comment by nehaljwani** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 
 package:
     name: libuuid
     version: {{ version }}
 
 source:
     fn: libuuid-{{ version }}.tar.gz
-    url: https://sourceforge.mirrorservice.org/l/li/libuuid/libuuid-{{ version }}.tar.gz
-    sha256: 46af3275291091009ad7f1b899de3d0cea0252737550e7919d17237997db5644
+    url: https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v2.32/util-linux-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipe/meta.yaml]: The hardcoded v2.32 here won't work well for the not.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,18 +1,22 @@
-{% set version = "1.0.3" %}
+{% set version = "2.32.1" %}
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipe/meta.yaml]: @nehaljwani, @jjhelmus, and @msarahan are you pinning `libuuid` in AnacondaRecipes?

[the remark this answers is not in the document]
Comment (jjhelmus) [reply recipe/meta.yaml]: We are not pinning but the package does have a run_export to the major version, https://github.com/AnacondaRecipes/libuuid-feedstock/blob/master/recipe/meta.yaml#L15-L17. Rebuilding downstream packages will be needed if the version is changed to 2.x as the current requirements are "libuuid >=1.0.3,<2.0a0" or similar.

[the remark this answers is not in the document]
Comment (jjhelmus) [reply recipe/meta.yaml]: The ABI for libuuid seems to be quite good in the 2.x line, https://abi-laboratory.pro/index.php?view=timeline&l=util-linux.  The less than 100% backwards compatibility seem to only be for other libraries.

[the remark this answers is not in the document]
Comment (nehaljwani) [reply recipe/meta.yaml]: I think we should bump the pinning to the 2.x line. Event CentOS 6 ships the 2.17.2 version.

BTW, I faced #15 when I compiled my own conda package for vim. (libSM.so (system lib) kept pointing to libuuid.so and wanted a newer version)

File: recipe/meta.yaml (hunk 1 above)
Comment (nehaljwani) [reply recipe/meta.yaml]: Fixed.

File: recipe/meta.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (CJ-Wright) [reply recipe/meta.yaml]: Attn: @scopatz

[the remark this answers is not in the document]
Comment (CJ-Wright) [reply recipe/meta.yaml]: If the number of recipes is medium to large, let me know and we can put up a rebuild migratior for this with the bot.

[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipe/meta.yaml]: The list I have is:
```
ossuuid
pynio
libhdfs3
fontconfig
xorg-libsm
postgres-xl
lighttpd
czmq
```

What worries me is `fontconfig` b/c many packages depend on it and may be linking against `libuuid` indirectly and must be re-build too.

**Comment by ocefpaf** [comment]
@nehaljwani we have tons of migration PRs open at the moment so I'll postpone this for the weekend when I can probably get more CI time to rebuild those quicker. Is that OK?

**Comment by nehaljwani** [comment]
No hurry from my side!

**Comment by ocefpaf** [comment]
Thanks @nehaljwani!

**Pull request merged**